connectors/sources/jira.py

```diff
@@ -688,6 +704,7 @@ async def get_content(self, issue_key, attachment, timestamp=None, doit=False):
         ):
             return
 
+        self._logger.info(f"Downloading content for file: {filename}")
```

Comment:
```suggestion
        self._logger.debug(f"Downloading content for file: {filename}")
```

nit. My rule-of-thumb is that anything that might get printed >10 times in a minute should probably be DEBUG